newParser.cpp

```diff
@@ -222,13 +219,18 @@ std::string commonItems::getNextLexeme(std::istream& theStream)
 				break;
 			}
 		}
-		else if (inputChar == '\n')
+		else if (!inString && inputChar == '\n')
```

Comment:
I tested and this caused no problems in Vic2 to HoI4, so it's good

Reply:
good, though I fear the strings will still retain \t padding after line breaks. Will fix this when I find a relevant test case (the names  thta prompted this fix were dropped during conversion process).